### ISSUE TITLE
http://ows.irill.org/latest/today/index.html 404s

### ISSUE THREAD

**Issue by rickyvetter** [OCamlPro/ows#17]
Not sure if this is intentional. If this is meant to be run locally then it'd be good to remove the link from the description of the repo.

**Comment by abate** [comment]
I don't think it is intentional. The problem is that the development of dose and opam diverged and the current version of ows (based on dose tools) uses an ancient version of opam. At some stage (unless the ows infrastructure is updated), this service will stop to exist.

**Comment by rickyvetter** [comment]
I see. So is the site down an indication that this end stage has been reached, or just a temporary blip that we can expect to come back up in the near future? I'm asking in relation to the linked issue to understand if the link should be removed from https://ocaml.org/community/support.html?

**Comment by abate** [comment]
I've tried to resurrect the service (I'm the original developer of ows), but I think the development of opam and the related changes to the repository have now gone too far. I think that you can remove the link for the moment to maybe add it back in the future.

**Comment by AltGr** [comment]
@abate thanks for the effort! I'll see if I can do something. I wonder if we couldn't have the old version still work, at least while the official repository is still in 1.2 format ?
Also, there are new fields like `conflict-class:` in the opam format, but I retained the PEF export, and they should be handled. Please tell if there is something specific you need.

**Comment by abate** [comment]
@AltGr maybe I declared it lost too early. I'll have a look, but don't hold your breath. Dose also moved away from the pef format in the mean time : the idea was to abandon the pef export that works for a specific switch at a time and to move to the new format that is multi-switch aware.

**Comment by AltGr** [comment]
Ok, thanks!

**Comment by rickyvetter** [comment]
I see that the index page is back up! Looks great. Most of the links in the main menu are still having some issues.

**Comment by abate** [comment]
yup. I worked a bit on it. In the end the problem is not so much opam related, but has more to do with the python glue I wrote to create the html pages. I kinda crippled the site to be able to get it to work again. I could just throw more RAM at it, or rework the data structures I'm using for the parsing. Anyway, not everything is lost... I'll keep you posted here.
